(fix) e2e: get-profile-activity test expects urn but FeedPost has url

## Problem

`packages/e2e/src/get-profile-activity.e2e.test.ts` line 137 expects `post.urn` but the `FeedPost` type has `url`, not `urn`.

## Evidence

```
AssertionError: expected { url: '', authorName: null, …(10) } to have property "urn"
❯ src/get-profile-activity.e2e.test.ts:137:22
```

Only the CLI `--json` test fails (1/3). The MCP tool test passes because it doesn't check individual post properties.

## Root Cause

The test was written expecting a `urn` field, but `FeedPost` (at `packages/core/src/types/feed.ts:9`) uses `url: string`. The `GetProfileActivityOutput.posts` is typed as `FeedPost[]`.

## Fix

Update the test assertion from `expect(post).toHaveProperty("urn")` to `expect(post).toHaveProperty("url")` at line 137.

## Acceptance Criteria

- [ ] All 3 tests in `get-profile-activity.e2e.test.ts` pass
- [ ] Test assertions match actual `FeedPost` type shape

--- a/packages/e2e/src/get-profile-activity.e2e.test.ts
+++ b/packages/e2e/src/get-profile-activity.e2e.test.ts
@@ -134,7 +134,7 @@ describeE2E("get-profile-activity operation", () => {
 
       // Profile may or may not have recent posts
       for (const post of parsed.posts) {
-        expect(post).toHaveProperty("urn");
+        expect(post).toHaveProperty("url");
         expect(typeof post.reactionCount).toBe("number");
         expect(typeof post.commentCount).toBe("number");
         expect(typeof post.shareCount).toBe("number");

